tests/core/mountpoint/test_base.py

```diff
@@ -55,26 +62,24 @@ async def test_mount_unknown_workspace(base_mountpoint, alice_user_fs, event_bus
 async def test_base_mountpoint_not_created(base_mountpoint, alice_user_fs, event_bus):
     # Path should be created if it doesn' exist
     base_mountpoint = base_mountpoint / "dummy/dummy/dummy"
-    mountpoint = f"{base_mountpoint.absolute()}/w"
 
     wid = await alice_user_fs.workspace_create("w")
     workspace = alice_user_fs.get_workspace(wid)
     await workspace.touch("/bar.txt")
 
-    bar_txt = trio.Path(f"{mountpoint}/bar.txt")
-
     # Now we can start fuse
 
     async with mountpoint_manager_factory(
         alice_user_fs, event_bus, base_mountpoint
     ) as mountpoint_manager:
-
         await mountpoint_manager.mount_workspace(wid)
+        bar_txt = get_path_in_mountpoint(mountpoint_manager, wid, "/bar.txt")
         assert await bar_txt.exists()
 
 
 @pytest.mark.trio
 @pytest.mark.mountpoint
+@pytest.mark.skipif(os.name == "nt", reason="Windows uses drive")
```

Comment:
Is there a test "drive already used" for windows ?

Reply:
I'll add one!